### PR TITLE
describe network

### PR DESCRIPTION
**Pull request by lschellenberg**
Calls chainhead and participation endpoint of Beacon API and prints relevant details

lukso network describe -beaconapi [BASE_URL](Default is hardcoded -> should be in node_config) -epoch [INT64] (default head epoch)

### CODE REVIEW

File: src/network/network_describe.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	"strconv"
+)
+
+func DescribeNetwork(baseUrl string, epoch int64) error {
+	client := beaconapi.NewBeaconClient(baseUrl)
+	chainHeadResponse, err := client.ChainHead()
+	if err != nil {
+		return err
+	}
+
```
Comment (atif-konasl) [inline src/network/network_describe.go:15]: can be nil checking before using `chainHeadResponse` for safety

File: src/network/network_describe.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+			return err
+		}
+	}
+
+	participationResponse, err := client.Participation(headEpoch)
+	if err != nil {
+		return err
+	}
+
+	utils.ColoredPrintln("Participation Epoch", participationResponse.Epoch)
```
Comment (atif-konasl) [inline src/network/network_describe.go:35]: same as before

**Review by atif-konasl — CHANGES_REQUESTED** [review]

File: src/network/network_describe.go (hunk 1 above)
Comment (lschellenberg) [reply src/network/network_describe.go:15]: not sure what u mean....

**Review by atif-konasl — CHANGES_REQUESTED** [review]

File: src/network/network_describe.go (hunk 1 above)
Comment (atif-konasl) [reply src/network/network_describe.go:15]: resolved

**Review by atif-konasl — APPROVED** [review]

**Pull request merged**